addons/zeusHC/functions/fnc_getSpawnMachineId.sqf

```diff
@@ -1,6 +1,8 @@
 #include "script_component.hpp"
 TRACE_1("Params",_this);
-params [["_allHCs", entities "HeadlessClient_F"]];
```

Comment:
SQF can be pretty dumb and won't do things like short-circuit eval unless you make it.
Even if you pass in a value for `_allHCs` in the params, arma is still going to evaulate the expression for the default value.

Here's an example:
```
getEntities = {
    systemChat "computing getEntities"; 
    [1]
};

[[5]] call {
    params [["_test", call getEntities]];
    systemChat str _test
}
```
Will output "computing getEntites - [5]".

So this is no worse than before